input/full.go

```diff
@@ -20,23 +19,14 @@ import (
 
 // CollectFull - Collects a "full" snapshot of all data we need on a regular interval
 func CollectFull(ctx context.Context, server *state.Server, connection *sql.DB, globalCollectionOpts state.CollectionOpts, logger *util.Logger) (ps state.PersistedState, ts state.TransientState, err error) {
-	systemType := server.Config.SystemType
 	ps.CollectedAt = time.Now()
 
-	ts.Version, err = postgres.GetPostgresVersion(ctx, logger, connection)
+	h, err := postgres.NewCollectionHelper(ctx, logger, server, globalCollectionOpts, connection)
```

Comment:
I still like just `postgres.NewCollection`, as I mentioned in PR Review. `Helper` doesn't really say anything, and conceptually I think it makes sense that a Collection is what eventually builds a Snapshot.